.tugboat/config.yml

```diff
@@ -0,0 +1,69 @@
+services:
+
+  mysql:
+    image: tugboatqa/mysql:5
+
+  app:
+    image: tugboatqa/php:7.2-apache
+    # Set this as the default service. This does a few things
+    #   1. Clones the git repository into the service container
+    #   2. Exposes port 80 to the Tugboat HTTP proxy
+    #   3. Routes requests to the preview URL to this service
+    default: true
+    depends: mysql
+    visualdiffs:
+      :default:
+        - /
+        - /test
+        - /blog/example
+        - /events
+
+    # A set of commands to run while building this service
+    commands:
+      # Commands that set up the basic preview infrastructure.
+      init:
+        # Tooling.
+        - apt-get update
+        - apt-get install -y libbz2-dev
+        - docker-php-ext-install opcache bz2 zip
+        - a2enmod headers rewrite
+        - wget -O /usr/local/bin/drush https://github.com/drush-ops/drush-launcher/releases/download/0.6.0/drush.phar && chmod +x /usr/local/bin/drush
+        # @todo remove jq once upstreams are all using govcms satis.
+        - wget -O /usr/local/bin/jq "https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64" && chmod +x /usr/local/bin/jq
+
+        # Clean up. This keeps image storage tighter on Tugboat.
+        - apt-get autoclean && apt-get clean
+
+      # Commands when refreshing an existing preview.
+      update:
+        # We have no external assets to update for testing the distro.
+
+      # Commands when building a new branch/PR update.
+      build:
+        # Add a "tugboat-latest" branch we can reference via composer in our /var/govcms/ project.
+        - git branch -D tugboat-latest || true
+        - git checkout -b tugboat-latest
+
+        # Tell the GovCMS project to use this "tugboat-latest" tag and the GovCMS satis.
+        # @todo remove references to satis once upstreams are all using satis.
+        - rm -Rf /var/govcms
+        - composer create-project -n --no-install govcms/govcms8-project /var/govcms
+        # Add local and hence latest govcms/govcms as repo source.
+        - composer --working-dir=/var/govcms config repositories.local path "${TUGBOAT_ROOT}"
+        - composer --working-dir=/var/govcms require zaporylie/composer-drupal-optimizations  govcms/govcms:dev-tugboat-latest
```

Comment:
I'd be happy to have `zaporylie/composer-drupal-optimizations` in the main composer.json for all builds - as per #306